src/test/java/com/epam/tc/nitcenkov/hw1/CalculatorTestAddition.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.tc.nitcenkov.hw1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculatorTestAddition {
+
+    @Test(dataProvider = "testAdditionDataProvider", groups = {"addsub"})
+    public void testAddition(double a, double b, double expected) {
+        Calculator calculator = new Calculator();
```

Comment:
Please create one parent class and move calculator initialization in `@Before...` method of the newly created class